tzrec/modules/enhanced_embedding.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2025, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from tzrec.datasets.utils import Batch
+from tzrec.modules.embedding import EmbeddingGroup
+
+
+class EnhancedEmbeddingGroup(nn.Module):
```

Comment:
Why should we use an EnhancedEmbeddingGroup? Is it possible to perform batch normalization and layer normalization within the EmbeddingGroup, or during the input initialization phase `init_input`?